selfdrive/locationd/laikad.py

```diff
@@ -449,7 +449,7 @@ def main(sm=None, pm=None):
   laikad = Laikad(save_ephemeris=not replay, auto_fetch_navs=use_internet, use_qcom=use_qcom)
 
   while True:
-    for in_msg in messaging.drain_sock(raw_gnss_sock):
+    for in_msg in messaging.drain_sock(raw_gnss_sock, wait_for_one=True):
```

Comment:
this makes it such that the clocks block below will never be hit. for this case, you probably just want to do a single blocking receive

Reply:
You mean because of this
```
 laikad.got_first_gnss_msg
 ```
 ?

Reply:
yeah

Reply:
Not fully sure why the clocks block is there, seems redundant

Reply:
Let me just remove